.github/workflows/checks.yml

```diff
@@ -13,7 +13,7 @@ jobs:
             - name: 'Install dependencies'
               run: npm install
             - name: 'Run linter'
-              run: npm run lint
+              run: ESLINT_USE_FLAT_CONFIG=false npm run lint
```

Comment:
Thanks. I'll try to migrate the configuration file in another PR, unless you plan to do it.

Reply:
It would be great if you could do it. Thanks.